gradle/libs.versions.toml

```
@@ -34,18 +34,18 @@ metalavaGradle = "me.tylerbwong.gradle:metalava-gradle:0.1.9"
 google-ksp-plugin = { module = "com.google.devtools.ksp:com.google.devtools.ksp.gradle.plugin", version.ref = "ksp" }
 google-ksp-api = { module = "com.google.devtools.ksp:symbol-processing-api", version.ref = "ksp" }
 
-androidx-core = "androidx.core:core-ktx:1.10.1"
-androidx-annotation = "androidx.annotation:annotation:1.6.0"
-androidx-fragment = "androidx.fragment:fragment-ktx:1.6.0"
-androidx-activity = "androidx.activity:activity:1.7.2"
-androidx-activity-compose = "androidx.activity:activity-compose:1.7.2"
+androidx-core = "androidx.core:core-ktx:1.12.0"
+androidx-annotation = "androidx.annotation:annotation:1.7.0"
+androidx-fragment = "androidx.fragment:fragment-ktx:1.6.1"
+androidx-activity = "androidx.activity:activity:1.8.0"
```

```suggestion
androidx-activity = "androidx.activity:activity:1.8.1"
```

Yesterday's fix related to `OnBackPressedDispatcher`